cql3/statements/select_statement.cc

```diff
@@ -304,7 +304,7 @@ select_statement::do_execute(service::storage_proxy& proxy,
 
     validate_for_read(cl);
 
-    int32_t limit = get_limit(options);
+    int64_t limit = get_limit(options);
```

Comment:
get_limit returns uint64_t - would it make sense to define limit as unsigned as well?